examples/stop-handler/stop_handler.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.runtime;
+
+string[] fruitBasket = ["Apple", "Orange"];
+
+function init() {
+    io:println("initial items in fruit basket: " + fruitBasket.toString());
+
+    // Registers a function that will be called during graceful shutdown.
+    runtime:onGracefulStop(stopHandlerFunc);
+}
+
+public function stopHandlerFunc() returns error? {
```

Comment:
Shall we rename the function as `clearBasket()` or similar? So that it will be consistent with the context.